Source/Factors/OktaFactorTotp.swift

```diff
@@ -25,13 +25,6 @@ open class OktaFactorTotp : OktaFactor {
                     onError: onError)
     }
 
-    public func activate(passCode: String,
```

Comment:
I would leave it, as `passCode` parameter is not optional here and shows that it must present to successfully activate TOTP factor

Reply:
Done.